source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure.Core/Cim/ValidationErrors/CimValidationErrorTextTemplateMessages.cs

```diff
@@ -217,6 +217,10 @@ public static class CimValidationErrorTextTemplateMessages
         public const string PriceSeriesResolutionMustMatchChargeResolutionErrorText =
             "The price series with transaction id {{ChargeOperationId}} contains a resolution {{ChargeResolution}} that does not match the charge with ID {{DocumentSenderProvidedChargeId}} of type {{ChargeType}} owned by {{ChargeOwner}}.";
 
+        [ErrorMessageFor(ValidationRuleIdentifier.PointsIsRequired)]
+        public const string PointsIsRequiredErrorText =
+            "Price series with transaction id {{ChargeOperationId}} does not contain any series period with positions for charge with ID {{DocumentSenderProvidedChargeId}} of type {{ChargeType}} owned by {{ChargeOwner}}.";
```

Comment:
Think this message is too tight coupled with CIM. The back office staff who'll be reading this will likely be better of with this?

```suggestion
            "Price series with transaction id {{ChargeOperationId}} does not contain any prices for charge with ID {{DocumentSenderProvidedChargeId}} of type {{ChargeType}} owned by {{ChargeOwner}}.";
```

Reply:
fixed